src/components/sidebar/index.tsx

```diff
@@ -120,10 +120,12 @@ const Sidebar = ({
 				{sidebarContent}
 				<SidebarHorizontalRule />
 				<ul className={s.navList}>
-					{generateResourcesNavItems(currentProduct.slug).map((item, index) => (
-						// eslint-disable-next-line react/no-array-index-key
-						<SidebarNavMenuItem item={item} key={index} />
-					))}
+					{generateResourcesNavItems(currentProduct?.slug).map(
```

Comment:
🌸 note to self, check in on what this is. See if there's a better way to handle this besides the optional chaining.

Reply:
💭 I'm hoping we'll see some DX improvement here with a task I have scheduled for this week, so it's helpful to see your note to self call this out. Task is: [[Sidebar] Make Resources section customizable by product](https://app.asana.com/0/1202097197789424/1202661920988657/f)